platforms-wg/glossary/_index.md

```diff
@@ -5,44 +5,62 @@ description: "A collection of terms used in the Platform Working Group published
 
 See also <https://glossary.cncf.io/>.
 
-A **platform** aggregates capabilities to serve developers and operators in
+## Platform
+A platform aggregates capabilities to serve developers and operators in
 development and delivery of products, services and apps. In reference to the
 scenarios it aims to support, a platform may be named a "Developer Platform", a
 "Delivery Platform", an "App Platform" or even a "Cloud Platform." The
 connotations of the older term "Platform-as-a-Service", or PaaS, are also
 influential.
+_See also [Platforms](#platforms)_
 
-**Platforms** enable developers and operators to deliver applications and
-services faster by providing and managing common capabilities. Platforms bridge
-between platform users and platform capability providers, and are built and
-maintained by platform teams.
+## Platforms
+Platforms enable developers and operators to deliver applications and
+services faster by providing and managing common capabilities. It is common for
+organisations to have more than one platform though it can be useful to maintain
+similar user experience across them.
+_See also [Platform](#platform)_
 
-**Platform capability providers** develop and maintain the capabilities offered
+## Platform capability providers
+The companies and people who develop and maintain the capabilities offered
 by the platform. Providers can be both external organizations or internal teams,
 and capabilities can be infrastructure, runtime, or other supporting services.
```

Comment:
I think we should say "The **projects and systems that provide the core capabilities offered** by the platform. Providers can be maintained by either external organizations or internal teams...."

While _people_ do build these projects and systems ultimately the system works without people. You could think of this as - people's involvement is incidental, not required.